haystack/nodes/retriever/sparse.py

```diff
@@ -17,6 +17,13 @@
 logger = logging.getLogger(__name__)
 
 
+try:
+    from sklearn.feature_extraction.text import TfidfVectorizer
+except ImportError as exc:
+    logger.debug("sklearn could not be imported. Run 'pip install farm-haystack[metrics]' to fix this issue.")
```

Comment:
It doesn't make much sense from the user perspective that they need to install `metrics` for TfidfVectorizer. And scikit-learn is not in the `metrics` anyway. It's part of the standard dependencies

Reply:
Uh right these should me removed. Thank you!